nativelib/src/main/scala/scala/scalanative/runtime/ieee754tostring/ryu/RyuDouble.scala

```diff
@@ -694,7 +696,9 @@ object RyuDouble {
 
   // format: on
 
-  @noinline def doubleToString(
+  // @deprecated("Use doubleToChars() instead, as it requires no additional String/Array allocations", "0.5.0")
```

Comment:
Yes leaving a "usage" comment here.

Personal opinion, I would not `@deprecate`, if for no other reason that we have to create entries in the 0.5.0 release notes file. There is heavy contention for that file, so it means weeks of waiting and having to come back in a few years and actually do
the removal.  Who said that, not me!

I have several such deprecation announcements & removals in my backlog, and I am dreading having to do them.  Not to spread discouragement, but to focus limited developer time.

Something like "Consider using doubleToChars() in new development.   mumble."

Reply:
|> So, should I let this line like this?

Sorry, I do not understand.  Could you rephrase?   Thanks.

Reply:
I mean, do I let the following comment line as is?
`// @deprecated("Use doubleToChars() instead, as it requires no additional String/Array allocations", "0.5.0")`

Or should I rephrase this?